src/demo-app/a11y/grid-list/grid-list-a11y.html

```diff
@@ -0,0 +1,42 @@
+<section>
+  <h2>Fixed-height grid list</h2>
+  <md-grid-list [cols]="fixedCols" [rowHeight]="fixedRowHeight">
```

Comment:
With the list roles removed from the components, should we add them to the demos for these cases that are non-interactive content?